Forwarded: not-needed
Last-Update: 2015-08-21
Author: Nikolaus Rath <Nikolaus@rath.org>
Description: Force use of libattr1-dev API
 Compilation under GNU/kFreeBSD fails because none of upstream's
 preprocessor macros matches this combination. It is also not clear
 how to properly test for the extended attribute API without 
 introducing something heavyweight like autoconf. In Debian, however,
 the libattr1-dev API is available for all supported architectures,
 so it is safe to just hardcode the use of this API. This is what
 this patch does.
 .

--- a/src/llfuse/capi.c
+++ b/src/llfuse/capi.c
@@ -7,14 +7,4 @@
 the terms of the GNU LGPL.
 */
 
-#ifdef __gnu_linux__
 #include "capi_linux.c"
-#elif __FreeBSD__
-#include "capi_freebsd.c"
-#elif __NetBSD__
-#include "capi_freebsd.c"
-#elif __APPLE__ && __MACH__
-#include "capi_darwin.c"
-#else
-#error "Unable to determine system (Linux/FreeBSD/Darwin)"
-#endif
